### DISCUSSION TITLE
golangci-lint in workflow in action failed to start

### DISCUSSION THREAD

**Discussion by robotLJW** [golangci/golangci-lint#1978] [Q&A] [answered]
In a Github CI build, I hit the issue of golangci-lint. The golangci-lint job displays startup failure，**but the job in  fork project can run successfully.** 
Please help take a look at it. Any directions would be appreciated. Thanks.

More detailed log is at https://github.com/apache/servicecomb-service-center/actions/runs/838057471
> golangci/golangci-lint-action@v2 is not allowed to be used in apache/servicecomb-service-center. Actions in this workflow must be: within a repository owned by apache, created by GitHub, verified in the GitHub Marketplace or match the following: */*@[a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9]+, AdoptOpenJDK/install-jdk@*, al-cheb/configure-pagefile-action@*, apache/*, burrunan/gradle-cache-action@*, bytedeco/javacpp-presets/.github/actions/*, chromaui/action@*, codecov/codecov-action@*, conda-incubator/setup-miniconda@*, container-tools/kind-action@*, dawidd6/action-download-artifact@*, gr2m/twitter-together@*, gradle/wrapper-validation-action@*, julia-actions/julia-runtest@*, julia-actions/setup-julia@*, msys2/setup-msys2@*, peaceiris/actions-gh-pages@*, peaceiris/actions-hugo@*, peter-evans/create-pull-request@*, potiuk/cancel-workflow-runs@*, r-lib/actions/*, ruby/setup-ruby@*, scacap/action-surefire-report@*, shivammathur/setup-php@*, shogo82148/actions-setup-perl@*...

The golangci-lint workflow in project  https://github.com/apache/servicecomb-service-center/actions/workflows/golangci-lint.yml

The golang-lint workflow in my forked project https://github.com/robotLJW/servicecomb-service-center/actions/workflows/golangci-lint.yml

**Comment by ldez** [comment, marked as answer]
Hello,

your project has some limitations defined in the repository settings:
- https://docs.github.com/en/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository
- https://docs.github.com/cn/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository

You have to change those settings.

**Reply by robotLJW** [reply]
Thanks.
